mlflow/store/tracking/rest_store.py

```diff
@@ -350,3 +360,13 @@ def _paginate_list_experiments(self, view_type):
             if not experiments.token:
                 break
             page_token = experiments.token
+
+    def search_experiments(
+        self,
+        view_type=ViewType.ACTIVE_ONLY,
+        max_results=None,
+        filter_string=None,
+        order_by=None,
+        page_token=None,
+    ):
+        raise NotImplementedError("Not implemented yet")
```

Comment:
Do we need to define `search_experiments` again in the `DatabricksRestStore` if the intended implementation will be identical to `RestStore`?

Reply:
I added this to silence the following pylint error:

```
> pylint mlflow/store/tracking/rest_store.py
************* Module mlflow.store.tracking.rest_store
mlflow/store/tracking/rest_store.py:326:0: W0223: Method 'search_experiments' is abstract in class 'RestStore' but is not overridden (abstract-method)
```

pylint still thinks `RestStore.search_experiments` must be overridden.

Reply:
Ref: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/abstract-method.html

Reply:
Reported this issue in the pylint repository: https://github.com/PyCQA/pylint/issues/7124

Reply:
One of the pylint maintaners replied to my comment: https://github.com/PyCQA/pylint/issues/7124#issuecomment-1174461952

`RestStore.search_experiments` is still an abstract method since it only contains `raise NotImplementedError`.

Reply:
Can we raise `MlflowException`?